src/views/forms/TradeForm.tsx

```diff
@@ -243,7 +243,7 @@ export const TradeForm = ({
 
         {complianceStatus === ComplianceStatus.CLOSE_ONLY && (
           <AlertMessage type={AlertType.Error}>
-            <$Message>{complianceMessage}</$Message>
+            <span>{complianceMessage}</span>
```

Comment:
$Message uses flex row. but the text here is too long and breaks the style. changing to span fixes this.